### PR TITLE
Password

### PR DESCRIPTION
**Pull request by lena-aiu**
I made changes with the password and removed video_layout.rb file. please review it. @jrmcgarvey

### CODE REVIEW

File: app/controllers/orders_controller.rb [hunk 1]
```diff
@@ -80,6 +80,7 @@ def destroy
   # GET /customers
   # GET /customers.json
   def index1
+    byebug
```
Comment (jrmcgarvey) [inline app/controllers/orders_controller.rb:83]: oops, need to get rid of this byebug

**Review by jrmcgarvey — CHANGES_REQUESTED** [review]
Hi Elena -- you left a byebug in the code.  All is good except for that.  You would change this current branch to remove the byebug and push the new commit, and your change will be added to the PR.

**Comment by lena-aiu** [comment]
Hi John! I removed byebug. @jrmcgarvey

**Review by jrmcgarvey — APPROVED** [review]
Ok, looks good, I will merge

**Pull request merged**